src/enforcer/source/sbom/maven.rs

```diff
@@ -24,7 +24,7 @@ impl Generator for MavenGenerator {
         })
     }
 
-    fn highlight(&self, dependency: &Dependency) -> anyhow::Result<(Url, Range)> {
+    fn highlight(&self, _: &Dependency) -> anyhow::Result<(Url, Range)> {
```

Comment:
Your change is correct. But my code is not :grin: 

I left this in as a todo/reminder that this needs to be improved. Maybe adding a proper todo marker would be better.

The idea is that the actual dependency should be found and highlighted, if possible. If it's a transient dependency, then this direct dependency (which pulls in the transient dependency) should be selected. If none can be found, then having the whole section marked is ok.